lib/App/Yath/Options/Runner.pm

```diff
@@ -62,7 +62,7 @@ option_group {prefix => 'runner', category => "Runner Options"} => sub {
     option unsafe_inc => (
         description => "perl is removing '.' from \@INC as a security concern. This option keeps things from breaking for now.",
         env_vars    => [qw/PERL_USE_UNSAFE_INC/],
-        default     => 1,
+        default     => 0,
```

Comment:
This is part of your other PR, can you pull this out of this PR?

Reply:
sure will do, this will increase the test complexity but can work around it

Reply:
updated via ea721431